providers/tests/google/cloud/operators/test_translate.py

```diff
@@ -19,15 +19,27 @@
 
 from unittest import mock
 
+from google.api_core.gapic_v1.method import DEFAULT
+from google.cloud.translate_v3.types import automl_translation
```

Comment:
test is failing:
https://github.com/apache/airflow/actions/runs/11982463149/job/33410779346?pr=44295#step:7:5367
```
    return _bootstrap._gcd_import(name[level:], package, level)
providers/tests/google/cloud/operators/test_translate.py:23: in <module>
    from google.cloud.translate_v3.types import automl_translation
E   ImportError: cannot import name 'automl_translation' from 'google.cloud.translate_v3.types' (/usr/local/lib/python3.11/site-packages/google/cloud/translate_v3/types/__init__.py)
----- generated xml file: /files/test_result-providers_google-postgres.xml -----
```